ui-components/src/forms/Selector.tsx

```diff
@@ -0,0 +1,24 @@
+import React from "react"
+import Select from "react-select"
+
+export interface SelectorProps {
+  options?: { value: string; label: string }[]
+  closeMenu?: boolean
+  defaultValue?: { value: string; label: string }[]
+  multiSelect?: boolean
```

Comment:
Can we add a prop for an `onChange()` callback? We'll need to be able to trigger actions when the select is changed.

there's docs here: https://react-select.com/props#select-props

Reply:
Hey Austin, I just made a new commit but I maybe misunderstanding the onChange prop.  Can you take a look and see what I might be doing wrong.

Reply:
i think I might've led you astray. I'll add details below in a general comment so they're not lost when you resolve this thread.